### PR TITLE
Automated PR for PDDL-1.0

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/PDDL-1.0.xml [hunk 1]
```diff
@@ -0,0 +1,348 @@
+<spdx name="ODC Public Domain Dedication &amp; License 1.0" identifier="PDDL-1.0" osi-approved="false">
+  <urls>
+    <url>http://opendatacommons.org/licenses/pddl/1.0/</url>
+  </urls>
+  <license>
+    <title>
+      <p>Open Data Commons - Public Domain Dedication &amp; License (PDDL)</p>
+    </title>
+    <optional>
```
Comment (jlovejoy) [inline src/PDDL-1.0.xml]: I'm not sure the preamble should be optional - it wasn't before... ??

**Comment by jlovejoy** [comment]
I'd like to see how this will look in HTML page.  The formatting in the original license was pretty flat, might be nice to use more nesting for sub-sections for ease of reading.

File: src/PDDL-1.0.xml (hunk 1 above)
Comment (bradleeedmondson) [inline src/PDDL-1.0.xml]: Agree; I think it should not be optional.

We could reconsider if we empirically find versions of this in the wild that both include and exclude the preamble.

**Comment by pmadick** [comment]
Agree that the preamble should NOT be optional.

**Comment by jlovejoy** [comment]
fix, to remove optional tag

**Pull request merged**